### PR TITLE
Removed unnecessary code-block directives in various docs.

### PR DESCRIPTION
**Pull request by jdufresne**
Better follows internal "Writing documentation".

https://docs.djangoproject.com/en/3.0/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files

> Add `.. code-block:: <lang>` to literal blocks so that they get
> highlighted. Prefer relying on automatic highlighting using :: (two
> colons). This has the benefit that if the code contains some invalid
> syntax, it won’t be highlighted. Adding `.. code-block:: python`, for
> example, will force highlighting despite invalid syntax.

### CODE REVIEW

File: docs/ref/templates/builtins.txt [hunk 1]
```diff
@@ -439,9 +439,7 @@ Use of both ``and`` and ``or`` clauses within the same tag is allowed, with
 
     {% if athlete_list and coach_list or cheerleader_list %}
 
-will be interpreted like:
-
-.. code-block:: python
```
Comment (felixxm) [inline docs/ref/templates/builtins.txt]: Please revert this change, because `.. highlight:: html+django` is set at the top of this file, so a default highlighting is different for this file.

Comment (jdufresne) [reply docs/ref/templates/builtins.txt]: Good catch. Thanks for finding that. Fixed in the latest revision.

**Comment by felixxm** [comment]
@jdufresne Thanks :+1:
